lib/adiwg/mdtranslator/readers/iso19115_2/modules/module_format.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+require 'adiwg/mdtranslator/internal/internal_metadata_obj'
+
+module ADIWG
+   module Mdtranslator
+      module Readers
+         module Iso191152
+            module Format
+               @@formatXPath = 'gmd:MD_Format'
+               @@formatNameXPath = 'gmd:name//gco:CharacterString'
+               def self.unpack(xDistFormat, hResponseObj)
+                  intMetadataClass = InternalMetadata.new
+                  hFormat = intMetadataClass.newResourceFormat
+
+                  # MD Format (required)
+                  xFormat = xDistFormat.xpath(@@formatXPath)[0]
+
+                  if xFormat.nil?
+                     msg = 'WARNING: ISO19115-2 reader: element \'gmd:MD_Format\' '\
+                        'is missing in \'gmd:distributionFormat\''
+                     hResponseObj[:readerExecutionMessages] << msg
+                     hResponseObj[:readerExecutionPass] = false
+                     return hFormat
```

Comment:
I'd like some more clarity from Jonathan about when to return nil vs an empty metadata object.

Reply:
i agree. i have this as an improvement in the -2 reader [ticket](https://github.com/GSA/data.gov/issues/4862#:~:text=determine%20which%20modules%20must%20return%20hash%20vs%20nil%20comment1%2C%20comment2)